### ISSUE TITLE
Adjustment for lnd.check.sh to not overwrite/delete externalip

### ISSUE THREAD

**Issue by myxmaster** [raspiblitz/raspiblitz#3688]
This might be rather a question than an issue, since I don't fully understand this.

With lines 184-190, this script (https://github.com/rootzoll/raspiblitz/blob/v1.8/home.admin/config.scripts/lnd.check.sh) checks if _runBehindTor_ is off. If so, it overwrites _externalip_ in lnd.conf with _${publicIP}:${lndPort}_ from /mnt/hdd/raspiblitz.conf otherwise **it deletes the line _externalip=*some-external-ip*_**.

I wonder if this is needed at all anymore. It seems to be more complicated than expected, looking at some possibly related stuff:
https://github.com/rootzoll/raspiblitz/issues/2787
-> is part of milestone 1.10.0 release https://github.com/rootzoll/raspiblitz/milestone/18
https://github.com/rootzoll/raspiblitz/issues/2855
https://github.com/rootzoll/raspiblitz/issues/3342

Anyway, when a user sets up a VPS to serve as forward and reverse proxy and makes LND understand this by inserting this:
```
externalip=VPS-PUBLIC-IP:PORT
nat=false

tor.active=true
tor.v3=true
tor.streamisolation=false
tor.skip-proxy-for-clearnet-targets=true
```
in the lnd.conf, then the lnd.check.sh would delete the `externalip=*VPS-PUBLIC-IP:PORT` after every reboot.

Long story short: Can we simply delete lines 184-190 or would that create other issues which I do not understand?

**Comment by rootzoll** [comment]
As you said .. its set to be something for a deeper focus with the later v1.10.0 milestone. Basically what you describe is when somebody has a fixed public IP to use for LND ... for now use the script `/home/admin/config.scripts/lnd.setaddress.sh on [YOUR-FIXED-IP]`to set your fixed IP.
